xml/admin_ceph_cephfs.xml

```diff
@@ -50,6 +50,20 @@
     <package>ceph-common</package> package (which is shipped with &sle;) should
     be installed before trying to mount &cephfs;.
    </para>
+   <important>
+     <para>
+       Without the <package>ceph-common</package> package (and thus without the
+       <command>mount.ceph</command> helper), the monitors IPs will need to
```

Comment:
```suggestion
       <command>mount.ceph</command> helper), the monitors' IPs will need to
```

Reply:
Does "monitors" here mean MON nodes? 

If so, I would suggest:

... the MON nodes' IP addresses must be used instead of their names.

Reply:
This is "monitors" :) https://www.kernel.org/doc/html/latest/filesystems/ceph.html#mount-syntax